FirstProject/evaluation.html

```diff
@@ -0,0 +1,85 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <link rel="stylesheet" href="css/main.css">
+        <link rel="stylesheet" href="img/">
+        <!-- Latest compiled and minified CSS -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+
+        <!-- jQuery library -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+        <!-- Latest compiled JavaScript -->
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    </head>
+
+    <body>
+        <div>
+            <div class="row">
+                <nav class="col-md-12" style="border-bottom: 2px solid darkgrey; padding-top: 12px; padding-bottom: 14px;">
+                    <div class="col-md-4" style="text-align:center;">
+                        <img href="#" style="width: 40%;" src="img/logo-v2.png"/>
+                    </div>
+                    <div class="col-md-4">
+                        <a href="#">Evaluations</a>
+                        <a href="#" style="margin-left: 100px;">New Evaluation</a>
+                    </div>
+                    <div class="col-md-4">
+                        <a href="#" style="float:right;">Logout</a>
+                    </div>
+                </nav>
+            </div>
+        </div>
+<br/>
```

Comment:
Things like `<br>` / `<b>` / `<i>` / `<u>` / `<hr>` should seldom (if at all) be used. In order to achieve proper separation of concerns, we'll want to delegate the styling of a page to the presentation layer (i.e., CSS), while keeping markup strictly for the data / model layer.

This makes it easier to affect just the look and feel of a page without changing the markup, or, conversely, modifying the information to be presented in a way that doesn't affect the overall look and feel. See the [MDN documentation](https://developer.mozilla.org/en/docs/Web/HTML/Element/br), for example, for more details.